qa/rpc-tests/wallet_protectcoinbase.py

```diff
@@ -157,16 +155,54 @@ def run_test (self):
         assert_equal("failed", status)
         assert_equal("wallet does not allow any change" in errorString, True)
 
+        # Add viewing key for myzaddr to Node 3
+        myviewingkey = self.nodes[0].z_exportviewingkey(myzaddr)
+        self.nodes[3].z_importviewingkey(myviewingkey, "no")
+
         # This send will succeed.  We send two coinbase utxos totalling 20.0 less a fee of 0.00010000, with no change.
         shieldvalue = Decimal('20.0') - Decimal('0.0001')
         recipients = []
         recipients.append({"address":myzaddr, "amount": shieldvalue})
         myopid = self.nodes[0].z_sendmany(mytaddr, recipients)
         mytxid = wait_and_assert_operationid_status(self.nodes[0], myopid)
         self.sync_all()
+
+        # Verify that z_listunspent can return a note which has zero confirmations
+        results = self.nodes[0].z_listunspent()
+        assert(len(results) == 0)
+        results = self.nodes[0].z_listunspent(0) # set minconf to zero
+        assert(len(results) == 1)
+        assert_equal(results[0]["address"], myzaddr)
+        assert_equal(results[0]["amount"], shieldvalue)
+        assert_equal(results[0]["confirmations"], 0)
```

Comment:
Should we check `assert_equal(results[0]["spendable"], True)` here with zero confirmations?

Reply:
Sure, I've added this, with a comment to clarify the note is unspent (which means its spendable) but it does need 1 confirmation before it can actually be spent.